.appveyor.yml

```diff
@@ -2,15 +2,17 @@
 clone_depth: 5                  # clone entire repository history if not defined
 
 environment:
-  CFLAGS: "--coverage"
-  LDFLAGS: "--coverage"
   TEST_SUITES: testinstall
+  # Test both 32-bit and 64-bit, and with and without built-in libraries
   matrix:
     - CYG_ARCH: x86
       CYG_ROOT: C:\cygwin
       ABI: 32
     - CYG_ARCH: x86_64
       CYG_ROOT: C:\cygwin64
+      PACKAGES: "-P libgmp-devel,zlib-devel"
+      CFLAGS: "--coverage"
+      LDFLAGS: "--coverage"
```

Comment:
So... why is --coverage not used for 32bit cygwin builds anymore?

Reply:
To increase testing, 64 bit uses external libraries, 32 bit uses internal libraries. zlib doesn't seem to build when we set cflags=--coverage.

Reply:
It looked like there was no 32bit specific cygwin code we need to make sure we cover.

Reply:
This really should be documented via comments, else this becomes unmaintainable down the road

Reply:
I updated the comment above; feel free to squash that added commit into your other commits, or edit the message further if you are not happy with it.